packages/amplify_auth_plugin_interface/lib/src/Password/ResetPasswordStep.dart

```diff
@@ -13,10 +13,10 @@
  * permissions and limitations under the License.
  */
 
-import 'package:flutter/foundation.dart';
+
 import '../types/AuthNextStep.dart';
 
 class ResetPasswordStep extends AuthNextStep {
   String updateStep;
-  ResetPasswordStep({additionalInfo, @required codeDeliveryDetails, @required this.updateStep}) : super(additionalInfo: additionalInfo, codeDeliveryDetails: codeDeliveryDetails);
+  ResetPasswordStep({required additionalInfo, required codeDeliveryDetails, required this.updateStep}) : super(additionalInfo: additionalInfo, codeDeliveryDetails: codeDeliveryDetails);
```

Comment:
Is this intended? `additionalInfo` was not required earlier?

Reply:
You're right it shouldn't need to be required especially because AuthNextStep superclass has a default value for additionalInfo. 

@haverchuck can you verify?

Reply:
Correct.